etc/config/ada.amazon.properties

```diff
@@ -29,6 +29,43 @@ compiler.gnatsnapshot.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 compiler.gnatsnapshot.name=x86-64 gnat (trunk)
 compiler.gnatsnapshot.semver=(trunk)
 
+################################
+# GNAT for riscv64
+group.gnatriscv64.compilers=gnatriscv64112:gnatriscv64103
+group.gnatriscv64.groupName=GNAT riscv64
+group.gnatriscv64.instructionSet=riscv
+group.gnatriscv64.baseName=riscv64 gnat
+group.gnatriscv64.isSemVer=true
+group.gnatriscv64.adarts=foo
+
+compiler.gnatriscv64103.exe=/opt/compiler-explorer/riscv64/gnat-arm-elf-linux64-10.3.0-2/bin/riscv64-elf-gnat
```

Comment:
```Jan/26 17:22 admin-node~/infra (main|✔) $ /opt/compiler-explorer/riscv64/gnat-arm-elf-linux64-10.3.0-2/bin/riscv64-elf-gnat
/opt/compiler-explorer/riscv64/gnat-arm-elf-linux64-10.3.0-2/bin/riscv64-elf-gnat: command not found                       
```

I think these need to be updated now with the new installation spot?

Reply:
yes probably. I tried to understand where they would end up, so this is my best guess with what has been merge in infra...

Reply:
It seems it needs to be:
```
Jan/26 18:28 admin-node~/infra (main|✔) 1.0s $ /opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-10.3.0-2/bin/riscv64-elf-gnat 
GNAT 10.3.0                                                                                                                          
Copyright 1996-2020, Free Software Foundation, Inc.                                                                                  
                                                                                                                                     
```